src/sentry/static/sentry/app/types/index.tsx

```diff
@@ -736,7 +736,7 @@ export type ProcessingIssue = {
   numIssues: number;
   signedLink: string;
   lastSeen: string;
-  hasMoreResolvableIssues: boolean;
+  hasMoreResolveableIssues: boolean;
```

Comment:
backend has a comment where they're keeping the spelling mistake for legacy api use. I don't believe the frontend is using it.

Reply:
Hmmmm, doesn't that mean the frontend uses the backend API response?

Reply:
Correct, the frontend uses this api, but this property didn't match the backend's spelling

Reply:
Ah I see